truss/util/.truss_ignore

```diff
@@ -161,7 +158,7 @@ cython_debug/
 #.idea/
 
 # Git
-.git/
+.git
```

Comment:
According to https://www.atlassian.com/git/tutorials/saving-changes/gitignore,  `.git` doesn't match the pattern `.git/`. I think we want `.git` because it includes the `.git` directory itself and all of its contents.